src/views/DashboardView.vue

```diff
@@ -0,0 +1,156 @@
+import MainNavbar from '../components/Navbar/MainNavbar.vue'
+import BoardsNavbar from '../components/Navbar/BoardsNavbar.vue'
+import EmptyInfo from '../components/EmptyInfo.vue'
+import BoardsColumn from '../components/BoardsColumn.vue'
+import DialogsTemplate from '../components/Dialogs/DialogsTemplate.vue'
+import { returnBoardProperties } from '../composables/boardProperties'
+import { returnDialogPropsToSelect } from '../composables/dialogHandling'
+import moreOptionsPopup from '../composables/moreOptionsPopup'
+import { ref, Ref, computed } from 'vue'
+import { useDark } from '@vueuse/core'
+
+const isDark = useDark()
+const isDashboardEmpty = ref(false)
+const boardProperties = returnBoardProperties()
+
+const areBoardOptionsShown = ref(false)
+const boardsNavbarProps = computed(() => ({
+  condition: windowWidth.value < 640 ? isNavOpen.value : isSidebarShown.value,
+  width: windowWidth.value,
+  theme: isDark.value,
+  boards: boardProperties.boards,
+  boardName: boardProperties.boardName,
+  toggleSidebar: toggleSidebar,
+  addNewBoard: () => toggleDialog('addBoard')
+}))
+const { toggleOptions, closeOptions } = moreOptionsPopup
+const callMoreOptionsFn = (e: Event, cb: (e: Event, conditionToChange: Ref<boolean>) => void) => {
+  cb(e, areBoardOptionsShown)
+}
+
+const propsToSelect = returnDialogPropsToSelect()
+const toggledDialog: Ref<ToggledDialog> = ref('seeTask')
+const isDialogOpen = ref(false)
+const dialogProps = computed(() => {
+  const sharedProps: SharedProps = {
+    isDark: isDark.value,
+    toggleDialog,
+    editTask: () => toggleDialog('editTask'),
+    editBoard: () => toggleDialog('editBoard'),
+    deleteTask: () => toggleDialog('deleteTask'),
+    deleteBoard: () => toggleDialog('deleteBoard'),
+    condition: isDialogOpen.value
+  }
+  const selectedProps = propsToSelect[toggledDialog.value as keyof PropsToSelect]
+
+  return { ...sharedProps, ...selectedProps }
+})
+const toggleDialog = (formType?: ToggledDialog) => {
+  if (formType) {
+    toggledDialog.value = formType
+  }
+  
+  if (formType === 'editTask' || formType === 'deleteTask') {
+    isDialogOpen.value = true
+    return
+  }
+  
+  isDialogOpen.value = !isDialogOpen.value
+}
+
+const isSidebarShown = ref(true)
+const isNavOpen = ref(false)
+const isLogoShown = ref(false)
+const toggleSidebar = () => {
+  isSidebarShown.value = !isSidebarShown.value
+  isLogoShown.value = false
+  setTimeout(() => {
+    if (!isSidebarShown.value) {
+      isLogoShown.value = true
+    }
+  }, 500)
+}
+const toggleBoardsNav = () => {
+  if (windowWidth.value >= 640) {
+    return
+  }
+  isNavOpen.value = !isNavOpen.value
+}
+
+const windowWidth = ref(window.innerWidth)
+const resizeObserwer = new ResizeObserver((entries) => {
+  windowWidth.value = entries[0].contentRect.width
+})
+resizeObserwer.observe(document.body)
+</script>
+
+<style scoped>
+.main-container {
+  @apply grid grid-rows-[80px_calc(100vh-80px)];
```

Comment:
Do we really need this calc?

Reply:
Yeah, again, I'm afraid so. This is still due to this animation I mentioned [here](https://github.com/Kepmon/task-manager/pull/1#discussion_r1214810002). Since I not only need my whole application ti fit within 100vw/100vh but also probably change the width of the right site of the app.

As I mentioned in the abovemention thread, I'll consider making it "a regular grid" back but first I need to figure out what to do with this animation.